package.json

```
@@ -31,7 +42,7 @@
     "url": "https://github.com/readmeio/oas/issues"
   },
   "scripts": {
-    "build": "tsc",
+    "build": "cross-env NODE_ENV=production tsup",
```

per my comment above:

```suggestion
    "build": "tsup",
```